package.json

```diff
@@ -83,9 +83,11 @@
     "@loadable/component": "^5.6.0",
     "@loadable/server": "^5.6.0",
     "@loadable/webpack-plugin": "^5.5.0",
+    "@storybook/addon-knobs": "^4.1.11",
     "@storybook/react": "^4.0.0-alpha.24",
```

Comment:
Worth looking at getting off the alpha while we're here? `4.1.11` is the latest atm.

Reply:
Consumers are already downloading latest because of the version range, but we should still do this anyway.